database/migrations/default/1744826895218_ems_remove_match_status_constraint/up.sql

```diff
@@ -23,25 +54,25 @@ BEGIN
     FROM people_list_view 
     WHERE id = NEW.person_id;
     
-    -- Update crah_pk of related EMS if not already matched to a crash
+    -- Update crash_pk of related EMS if not already matched to a person
     UPDATE ems__incidents 
     SET 
         crash_pk = matching_person_record.crash_pk,
         updated_by = NEW.updated_by,
         crash_match_status = 'matched_by_manual_qa'
     WHERE
-        incident_number = NEW.incident_number
+        id != NEW.person_id
```

Comment:
im very confused by this line, what does `ems__incidents.id` have to do with `NEW.person_id`?

Reply:
that's a bug! just changed to `NEW.id` 🙏